packages/node/src/methods/state-channel/deposit/controller.ts

```diff
@@ -34,22 +42,34 @@ export default class DepositController extends NodeController {
     const { store, provider } = requestHandler;
     const { multisigAddress, amount } = params;
 
+    params.tokenAddress = params.tokenAddress
```

Comment:
Don’t mutate the params object

Reply:
If this is not done here, this check needs to happen in the logic of the next 3 function calls. Also doesn't seem like we'd gain much by copying the params and modifying this member then passing that in.